azure-pipelines.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       SessionTimeout: '60'
       MaxConcurrency: '50'
       MaxRetryAttempts: '5'
-    condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['Build.SourceBranch'], 'refs/heads/release_4.0')))
+    condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['Build.SourceBranch'], 'refs/heads/release_4.0-hotfix-1')))
```

Comment:
May want to add `release_4.0-hotfix-1` to line 8

Reply:
Good catch @soninaren . This will be to trigger a pipeline run.